lib/core/connect.js

```diff
@@ -14,6 +14,12 @@
 var debug = require('debug')('keystone:core:connect');
 
 function connect() {
+	var warningMsg = 'keystone.connect() is now deprecated and will not be available in future versions of KeystoneJS\n' +
```

Comment:
I think we should actually throw here, since the `connect` function won't work at all in 0.3